api/src/utils/test/utils.ts

```
@@ -34,15 +46,147 @@ const extractInstances =
+
+  if (canAutoInjectFromAll || autoInjectFrom?.includes('controllers')) {
+    nestedProviders.push(...controllers, ...getNestedDependencies(controllers));
+  }
+
   const module = await Test.createTestingModule({
+    imports: [
+      ...(canInjectModels(imports)
+        ? [
+            MongooseModule.forFeature([
+              ...getModels(models),
+              ...(!!autoInjectFrom
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)